src/test/resources/test.sql

```diff
@@ -19,3 +20,4 @@ insert into otp_config (tenant_id, is_otp_mocked, whitelisted_inputs) values ('t
 insert into user_config (tenant_id, host, port, is_ssl_enabled, get_user_path, create_user_path, authenticate_user_path, add_provider_path) values ('tenant2', 'localhost', 9090, 0, '/user', '/user', '/authenticateUser', '/provider') ON DUPLICATE KEY UPDATE tenant_id = "tenant2";
+insert into contact_verify_config (tenant_id, is_otp_mocked, whitelisted_inputs) values ('tenant2', 0, '{}') ON DUPLICATE KEY UPDATE tenant_id = "tenant2";
```

Comment:
is_otp_mocked can be skipped as it has a default false value. Same for whitelisted_inputs, it has a default value.